### PR TITLE
Update functions for new schema and add new metrics

### PR DESCRIPTION
**Pull request by pacospace**
linked to https://github.com/thoth-station/storages/pull/702

### CODE REVIEW

File: thoth/metrics_exporter/jobs.py [hunk 1]
```diff
@@ -200,3 +141,77 @@ def get_python_packages_solver_error_count():
...
+        graph_db = GraphDatabase()
+        graph_db.connect()
+
+        total_unique_python_packages = len(graph_db.get_python_packages())
+
+        graphdb_total_unique_python_packages.set(total_unique_python_packages)
+
+        _LOGGER.debug("graphdb_total_unique_python_packages=%r", len(graph_db.get_python_packages()))
+
+    except aiohttp.client_exceptions.ClientConnectorError as excptn:
```
Comment (fridex) [inline thoth/metrics_exporter/jobs.py]: Do we encounter these exception types or is that a left-over from JanusGraph? I would say we now need to handle gRPC related exceptions as we use gRPC.

File: app.py [hunk 2]
```diff
@@ -73,17 +75,19 @@ def prepare_response(req, resp):
...
+        get_unique_run_software_environment_count()
+        get_user_unique_run_software_environment_count()
+        get_unique_build_software_environment_count()
 
     @api.background.task
     def update_openshift_metrics():
         _LOGGER.debug("updating OpenShift metrics")
 
-        get_thoth_solver_jobs('thoth-test-core')
+        print("No openshift metrics")
```
Comment (fridex) [inline app.py]: Forgotten print statement?

**Comment by fridex** [comment]
> linked to thoth-station/storages#702

BTW you can use "Depends-On: https://github.com/thoth-station/storages/pull/702" - in that case Zuul will wait until the PR in thoth-station/storages gets merged before merging this one.

File: app.py (hunk 2 above)
Comment (pacospace) [reply app.py]: I'm working on the service metrics, ```get_thoth_solver_jobs()``` needs to be rewritten

**Comment by pacospace** [comment]
Thanks Frido

**Pull request merged**